bokehjs/src/coffee/tool/inspectors/hover_tool.coffee

```diff
@@ -87,11 +86,14 @@ define [
 
       xmapper = frame.get('x_mappers')[renderer.mget('x_range_name')]
       ymapper = frame.get('y_mappers')[renderer.mget('y_range_name')]
-      x = xmapper.map_from_target(vx)
-      y = ymapper.map_from_target(vy)
+      # get x, y values from the rendered glyph
+      geom_x = xmapper.map_from_target(vx)
+      geom_y = ymapper.map_from_target(vy)
```

Comment:
they are not used (yet). No sense to use it if I'm following a wrong path. If, instead, what I've commented makes sense I'd suggest adding `geom_x` and `geom_y` or something similar [here](https://github.com/bokeh/bokeh/blob/feature/hover_along_lines/bokehjs/src/coffee/tool/inspectors/hover_tool.coffee#L134-L141)